### PR TITLE
waitress-serve: add more information link

### PR DESCRIPTION
**Pull request by lakhininenina**
Add "more information" link

- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
 
related to: #6062

### CODE REVIEW

File: pages/common/waitress-serve.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # waitress-serve
 
 > Pure Python WSGI HTTP Server.
+> More information: <https://docs.pylonsproject.org/projects/waitress/en/stable/runner.html>.
```
Comment (CleanMachine1) [inline pages/common/waitress-serve.md:4]:
```suggestion
> More information: <https://github.com/Pylons/waitress>.
```

Comment (navarroaxel) [reply pages/common/waitress-serve.md:4]: The project's website is <https://docs.pylonsproject.org/projects/waitress>. but the PR's author used a link to the pseudo man page

Comment (navarroaxel) [reply pages/common/waitress-serve.md:4]: @CleanMachine1 The project's website is [docs.pylonsproject.org/projects/waitress](https://docs.pylonsproject.org/projects/waitress). but the PR's author used a link to the pseudo man page

Comment (CleanMachine1) [reply pages/common/waitress-serve.md:4]: OK, either works for me!

Comment (navarroaxel) [reply pages/common/waitress-serve.md:4]: @CleanMachine1 so, do you dismiss your suggestion?

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**